ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.html

```diff
@@ -0,0 +1,27 @@
+<main class="participation">
+    <h2 class="title" translate>App.ParticipationPage.Title</h2>
+    <section *ngFor="let stepNumber of '12345'.split(''); let last=last;" class="step">
+        <div class="step__data">
+            <span class="step__label"
+                  [innerText]="translationStepData(stepNumber, '.Step') | translate">
+            </span>
+            <span class="step__time" *ngIf="!last"
```

Comment:
I'm not sure it's the best way to identify if we should display time or not. It's better to have a data structure array and check if the item contains the time. In case if in the future something will be changed and time will be removed from other items it will be harder to fix

Reply:
same for button

Reply:
Fixed at 9d6ef0c989ea9767fd5ef6d023506ef7eb753b1d